spec/models/work_spec.rb

```diff
@@ -50,6 +50,31 @@
     expect { work.save! }.to raise_error ActiveRecord::RecordInvalid
   end
 
+  describe "#editable_by?" do
+    subject(:work) { FactoryBot.create(:tokamak_work) }
+    let(:submitter) { User.find(work.created_by_user_id) }
+    let(:other_user) { FactoryBot.create(:princeton_submitter) }
+    let(:pppl_moderator) { FactoryBot.create(:pppl_moderator) }
+    let(:research_data_moderator) { FactoryBot.create(:research_data_moderator) }
```

Comment:
I was expecting to see a negative test for the research_data_moderator (since the work is in the PPL Collection)

Reply:
Added!